bin/01_train_classifier.py

```diff
@@ -5,13 +5,16 @@
 
 from traffic_cam import classifier, paths
 
+import os
+
 
 # data generators for training and validation
 train_generator = classifier.get_image_datagen(folder=paths.TRAIN_DIR, batch_size=16)
 valid_generator = classifier.get_image_datagen(folder=paths.VALID_DIR, batch_size=16)
 
 # train model
-model = classifier.get_classifier(n_classes=5)
+classes = os.listdir(paths.TRAIN_DIR)
+model = classifier.get_classifier(n_classes=len(classes))
 
```

Comment:
```suggestion
number_classes = len(os.listdir(paths.TRAIN_DIR))
model = classifier.get_classifier(n_classes=classes)
```
written this way, I would require less thinking :-)

Reply:
I thought we might need classes again, but... YAGNI!